src/main/java/racingcargame/InputView.java

```diff
@@ -18,4 +18,18 @@ public int inputCount() {
     public void close() {
         scanner.close();
     }
+
+    public String checkWordCount(String nameOfCars) throws WordCountException {
+        String[] carName = nameOfCars.split(",");
+        for (int i = 0; i < carName.length; i++) {
+            wordCountException(carName[i]);
+        }
```

Comment:
향상된 for문을 사용해주셔도 될 것 같네요! 
```suggestion
       for (String s : carName) {
            wordCountException(s);
        }
```